src/fbteken.c

```diff
@@ -1365,6 +1377,12 @@ main(int argc, char *argv[])
 		case 'o':
 			kbd_options = optarg;
 			break;
+		case 'p':
+			pivot = true;
+			break;
+		case 'P':
```

Comment:
I don't see any reason to have a flag for disabling this option, since it's disabled by default anyway.